lib/executor/ramping_arrival_rate.go

```diff
@@ -96,6 +96,7 @@ func (varc *RampingArrivalRateConfig) Validate() []error {
 	}
 
 	errors = append(errors, validateStages(varc.Stages)...)
+	errors = append(errors, validateTargetShifts(varc.StartRate.Int64, varc.Stages)...)
```

Comment:
Thought better on it and now I don't think we need it. The target in this case is related to iterations and not to VUs so having a high number here will not guarantee an insane number of VUs. In any case, for sure, the number of max iterations is in order of magnitude higher than the max allowed for VUs. Each VU can handle a rate of thousands of iterations.

At this point, it makes sense to just drop it from here. Having a static-constant value doesn't seem to be the right approach.
I'm sorry, I know I requested it in the previous review.

Instead, all the other executors that directly set the VUs number may also benefit from the same validation (shared-iterations, constant-vus, per-vu-iterations). But it is out-of-scope for this pull request, we will open a dedicated issue for them.